### DISCUSSION TITLE
서브 미팅 내용입니다!

### DISCUSSION THREAD

**Discussion by artemismars** [artemismars/mern_blog#14] [Announcements]
1. commit 메세지 컨벤션 통일.
    1. 예시: add, remove, error fix, restyle, make
2. 프론트팀 백엔드팀 컨벤션 팀별 조율
3. 프로젝트 코드에 대한 의견 공유 관련하여 깃헙 issues 활용하기로 결정 ( 팀에 의견전달할때 다른 수단이 아닌 issues로 의견전달 )
4. 태욱님 리액트 & 파이어베이스로 SNS 개인 프로젝트 배포 경험있으며 follower, comment, profile 기능 구현 경험.
5. 브랜치 구조는 `main` 이 최상위, `fe_dev`, `be_dev` 브랜치만 `main` 에 merge 될 수 있도록 설계.
6. 깃허브 Projects 랑 Issues 이용하기로 결정.
7. 깃허브 Projects에서 각자 기능 구현 목표치 할당하여 마일즈스톤 이용해서 기능 구현이 얼마나 완성되고 있는지 프로그레스바로 확인하기.

**Comment by donghyunami** [comment]
확인
